### DISCUSSION TITLE
How to Get a Supporter Certificate

### DISCUSSION THREAD

**Discussion by yijiangchengming** [sandboxie-plus/Sandboxie#1642] [Q&A Feedback] [answered]
I want to get a supporter certificate, but paypal doesn't work. The only payment method available is Alipay. Is it possible to increase Alipay payment.

**Comment by DavidXanatos** [comment, marked as answer]
You can send me crypt currency and get a certificate by email.
I don't have an alipay account, or similar only paypal.

**Reply by yijiangchengming** [reply]
That's a shame, I don't know a thing about cryptocurrencies.
